Pinta.Core/Managers/ChromeManager.cs

```diff
@@ -145,14 +145,23 @@ public void InitializeSimpleEffectDialog (SimpleEffectDialogHandler handler)
 		simple_effect_dialog_handler = handler;
 	}
 
-	public void ShowErrorDialog (Gtk.Window parent, string message, string body, string details)
-	{
-		error_dialog_handler (parent, message, body, details);
+	public async void ShowErrorDialog (
+		Gtk.Window parent,
+		string message,
+		string body,
+		string details)
+	{
+		string response = await error_dialog_handler (parent, message, body, details);
+		switch (response) {
+			case nameof (DialogResponses.Bug):
+				PintaCore.Actions.Help.Bugs.Activate ();
```

Comment:
The actual implementation of this is really simple (just a `system.LaunchUri()` call, so that could be moved somewhere else that's more convenient, if adding a dependency on Actions.Help is undesirable

I agree it shouldn't be pushed farther out than this, since the calling code doesn't care what options the error dialog's UI provides

Reply:
Yes, it could be as easy as this, but semantically speaking this has the disadvantage of repeating the (admittedly simple) implementation of the bug tracker launch in several places, so if it changed, we would have to update it in several places. What if we leave this for a future pull request?

Reply:
Yeah that's fine for a future PR.
I meant that we still would have only a single implementation, somewhere that both this location and `Actions.Help.Bugs.Activate()` could call into, so that updating the URL is easy if needed